lib/spack/spack/schema/config.py

```diff
@@ -87,6 +87,7 @@
                 "anyOf": [{"type": "integer", "minimum": 1}, {"type": "null"}]
             },
             "allow_sgid": {"type": "boolean"},
+            "install_status": {"type": "boolean"},
```

Comment:
I would just substitute "module_roots" below at line 96 with `terminal_title`  and say it has no effect

Reply:
Thanks for the hint, I completely missed this. I just pushed a new commit that replaces the old message with a new one for `terminal_title` (I hope that is what you meant).